05-classification.Rmd

```diff
@@ -511,7 +533,7 @@ library(tidymodels)
 
 Let's again suppose we have a new observation with perimeter 0 and concavity
 3.5, but its diagnosis is unknown (as in our example above). Suppose we 
-want to use the perimeter and concavity explanatory variables/predictors to
+want to use the perimeter and concavity predictor variables to
 predict the diagnosis class of this observation. Let's pick out our 2 desired
 predictor variables and class label and store it as a new dataset named `cancer_train`:
 
```

Comment:
Below we teach:
```
knn_fit <- knn_spec %>%
  fit(Class ~ ., data = cancer_train)
knn_fit
```

Should we instead do:

```
knn_fit <- knn_spec %>%
  fit(Class ~ Perimeter + Concavity, data = cancer_train)
knn_fit
```

So that they learn they can do this kind of syntax (which is more readable when the model is simple, and very commonly seen in R)? And then later, when we show them > 2 predictors move to the `Class ~ .` syntax?

Reply:
Also below, let's change:

"Finally, we make the prediction on the new observation by calling the `predict` function,passing the fit object we just created. "

to 

"Finally, we make the prediction on the new observation by calling the `predict` function,passing the fit object we just created and the new observation."

(sorry for not suggesting a change, GitHub wouldn't let me again...)

Reply:
I ended up just doing a little paragraph on `resp ~ pred + pred` vs ` resp ~ .` where I first introduce it. Makes sense to do introduce that concept here. Also if we don't do that we'll have to change everything downstream, which will certainly introduce new errors